src/main/java/com/google/sps/data/EntryObject.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EntryObject {
+  List<Object> hourList = new ArrayList<>();
+  List<Object> difficultyList = new ArrayList<>();
+  List<String> container = new ArrayList();
+
+  public EntryObject(String hourTitle, String diffTitle) {
+    List<String> container = new ArrayList();
+    container.add(hourTitle);
+    hourList.add(container);
+
+    container.removeAll(container);
+    container.add(hourTitle);
+    difficultyList.add(container);
+  }
+
+  public void addTitle(String title) {
+    List<String> container = new ArrayList();
```

Comment:
maybe name these titleContainer, hourContainer, etc. just so the number of variables named container doesn't confuse anyone?